report_py3o_signer/models/ir_actions_report.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Acsone SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+import os
+import tempfile
+from contextlib import closing
+
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+
+class IrActionReport(models.Model):
+
+    _inherit = "ir.actions.report"
+
+    def _check_valid_action_report(self):
+        res = super()._check_valid_action_report()
+        res += self.report_type != "py3o"
```

Comment:
+= on booleans is a bit weird. 

I'd say `return res or (self.report_type == "py3o")`.